src/OrbitGl/GraphTrack.cpp

```diff
@@ -92,6 +92,7 @@ void GraphTrack::Draw(GlCanvas* canvas, PickingMode picking_mode, float z_offset
   const Color kBlack(0, 0, 0, 255);
   const Color kWhite(255, 255, 255, 255);
   float text_z = GlCanvas::kZValueEvent + z_offset;
+  float label_z = GlCanvas::kZValueTrackText + z_offset;
```

Comment:
I'm not sure here. We are using the Event layer to draw text and the text layer to draw the label. I feel we need a new label for this elements which has to be on top of the text. I'm not sure about the name, could be simply kZValueTrackLabel and I will propose (in GlCanvas):
Event -> 0.07
TrackText -> 0.08
TrackLabel ->0.09

After that I would write here:
```
  float text_z = GlCanvas::kZValueTrackText + z_offset;
  float label_z = GlCanvas::kZValueTrackLabel + z_offset;
```

Reply:
Thanks for the suggestion and also explaining the purpose of different layers to me! I have changed the code accordingly.